supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/api/ActionBuilder.scala

```diff
@@ -18,30 +19,32 @@ import scala.concurrent.duration._
   * Utility singleton to create high level actions.
   */
 object ActionBuilder {
-  object Labels {
-    object Key {
-      /**
-        * Very short description of the action
-        */
-      val Tag = "tag"
-      val Logging = "logging"
-      val InputName = "inputName"
-      val OutputName = "outputName"
-    }
-    object Value {
-      val ContainerSetup = "ContainerSetup"
-      val UserAction = "UserAction"
-      val Localization = "Localization"
-      val Delocalization = "Delocalization"
-      val Background = "Background"
-      val RetryWithMoreMemory = "CheckingForMemoryRetry"
-    }
-  }
-
   implicit class EnhancedAction(val action: Action) extends AnyVal {
     private def javaFlags(flags: List[ActionFlag]) = flags.map(_.toString).asJava
 
+    /**
+      * Only for use with docker images KNOWN to not have entrypoints already set,
+      * or used with accompanying call to setEntrypoint("non-empty-string").
+      *
+      * Otherwise use withEntrypointCommand() below until the google issues listed in BA-6406 are fixed.
+      */
     def withCommand(command: String*): Action = action.setCommands(command.toList.asJava)
+
+    /**
+      * Useful for any externally provided images that _might_ have entrypoints already set.
+      * Hack for BA-6406. See underlying google issue in that ticket.
```

Comment:
Same comment: BA-6406 is "Done" so this should be considered "how we interact with PAPI" for the foreseeable future.